native_client/java/libdeepspeech/src/main/java/org/deepspeech/libdeepspeech/DeepSpeechModel.java

```diff
@@ -1,15 +1,37 @@
 package org.deepspeech.libdeepspeech;
 
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+
 /**
  * @brief Exposes a DeepSpeech model in Java
  **/
 public class DeepSpeechModel {
 
-    static {
+    /**
+    * Create a new DeepSpeechModel object and load the native libraries.<br>
+    * Use on: Android
+    */
+    public DeepSpeechModel() {
```

Comment:
This will change the API, what is wrong with `static {}` ?

Reply:
Static would always load the libraries that specific way. The way the libraries are loaded for Android don't play nice with standalone.

For end-developers nothing should change though, since the load libraries function is now called in the constructor, for if they dont specify the boolean nothing changes